Specs/DateTime/Portuguese/TimeExtractor.json

```diff
@@ -568,10 +568,10 @@
     "NotSupported": "javascript, python",
     "Results": [
       {
-        "Text": "de madrugada",
+        "Text": "madrugada",
```

Comment:
"madrugada" should be the early morning range.

Reply:
modified to use "meio da madrugada"